docs/current-1.1/tools/dotnet-nuget-push.md

```diff
@@ -0,0 +1,116 @@
+---
+title: dotnet-nuget-push command | .NET Core SDK
+description: The dotnet-nuget-push command pushes a package to the server and publishes it. 
+keywords: dotnet-nuget-push, CLI, CLI command, .NET Core
+author: karann-msft
+ms.author: mairaw
+manager: wpickett
+ms.date: 11/10/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: f54d9adf-94f8-41cc-bb52-42f7ca3be6ff
+---
+
+#dotnet-nuget-push
+
+[!INCLUDE[preview-warning](../../includes/warning.md)]
+
+## Name 
+dotnet-nuget-push -- Pushes a package to the server and publishes it. 
+
+## Synopsis
+
+`dotnet nuget push [<package>] [--help] [--source] [--symbols-source] 
+    [--timeout] [api-key] [--symbol-api-key] [--disable-buffering] [--no-symbols] 
+    [--force-english-output] [--config-file] [--verbosity]`
+
+## Description
+
+The `dotnet nuget push` command pushes a package to the server and publishes it. 
+The push command uses server and credential details found in the system's NuGet config file, or chain of config files (see <here> for more information on config files). 
```

Comment:
use this link for <here>
https://docs.nuget.org/ndocs/consume-packages/configuring-nuget-behavior